features/subscriber_can_comment_on_article.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Subscriber can comment on article
+
+  As a Subscriber
+  In order to voice my opinion or ask questions
+  I would like to be able to leave a comment on an article
+
+  Background:
+    Given the following user exists
+      | first_name | last_name    | email          | password | role       |
+      | William    | Schneiderman | will@gmail.com | password | member     |
```

Comment:
Again, the title suggests that a Subscriber can comment on articles. So does the user story....

Reply:
we have talked through the group and have decided that we should allow all users to comment.

Reply:
Yet you have not changed the user story. Also, making this kind of changes late in the process is not really okay. (not a blocker though)

Reply:
noted, please check again it should be pushed now